pkg/controllers/vdb/podannotatelabel_reconcile.go

```diff
@@ -29,23 +29,23 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-// PodAnnotationReconciler will maintain annotations in the pod about the running system
-type PodAnnotationReconciler struct {
+// PodAnnotateLabelReconciler will maintain annotations and labels in pods about the running system
+type PodAnnotateLabelReconciler struct {
```

Comment:
Minor question here Why did you change the name of this struct? And why is it PodAnnotateLabel and not something like PodAnnotationLabel or PodAnnotationAndLabel?

Reply:
This reconciler does more stuff now.  In addition to adding annotations to pods, it can add labels too.  I was trying to keep the name small.  I agree that the current name isn't ideal.  I'll change it to AnnotateAndLabelPodReconciler